app/src/main/java/a/a/a/Dp.java

```diff
@@ -169,7 +169,8 @@ public void a(iI iIVar, String str) {
      */
     public final void a(String outputPath, ArrayList<String> dexes) throws Exception {
         try {
-            if (!(Integer.parseInt(this.a) >= 21)) {
+            int minSdkVersion = Integer.parseInt(settings.getValue(ProjectSettings.SETTING_MINIMUM_SDK_VERSION, "21"));
```

Comment:
Use that upper Variable instead of hardcoaded "21"

Reply:
im not sure if that variable is not initialized in future, so i's rather use that hardcoded int